internal/rbd/rbd_journal.go

```diff
@@ -638,7 +644,7 @@ func RegenerateJournal(
 			return "", err
 		}
 
-		return rbdVol.VolID, nil
+		return rbdVol.VolID, err
```

Comment:
`err` will always be `nil` here. I prefer to have an explicit **no error** case, and returning `err` here make it look like there could be still an error lingering, in which case the `rbdVol.VolID` should not be trusted.